app/src/main/java/gitlin/kothub/MainActivity.kt

```diff
@@ -32,6 +34,8 @@ class MainActivity : AppCompatActivity(), AnkoLogger {
         super.onCreate(savedInstanceState)
         setContentView(activity_main)
         tv_hello.text = "Kothub"
+
+        ImageLoader.getInstance().init(ImageLoaderConfiguration.createDefault(this))
```

Comment:
Is this necessary with https://github.com/kothub/kothub/pull/10/files#diff-dd693a96b4910e148ca7517b2b9027f9R50 ?